emqx/README.md

```diff
@@ -0,0 +1,58 @@
+# Agent Check: EMQX
+
+## Overview
+
+This check monitors [EMQX][1] through the Datadog Agent in order to:
```

Comment:
Can you please provide a bit more context on:
- What EMQX is/what it does
- The type of data this integration sends into Datadog/why it's valuable for the user